components/sp-seamless-provision/src/main/java/com/wso2telco/carbon/identity/oauth2/token/handlers/clientauth/SeamlessProvisionClientAuthHandler.java

```diff
@@ -206,5 +218,23 @@ private MobileConnectConfig getMobileConnectConfig() {
         mobileConnectConfigs = configurationService.getDataHolder().getMobileConnectConfig();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AS2696)